classes/local/table/interaction_remaining_table.php

```diff
@@ -194,4 +194,18 @@ public function col_lastmodified($row) {
         return userdate($row->lastmodified, $dateformat);
     }
 
+    /**
+     * This function is not part of the public api.
+     */
+    public function print_nothing_to_display() {
+        global $OUTPUT;
+
+        // Render button to allow user to reset table preferences.
+        echo $this->render_reset_button();
```

Comment:
Although this has also been done in other tables classes of lifecycle, why is line 204 and 206 necessary for empty tables?